src/pykeen/training/slcwa.py

```diff
@@ -56,13 +55,10 @@ def __init__(
             optimizer=optimizer,
             automatic_memory_optimization=automatic_memory_optimization,
         )
-
-        if negative_sampler_cls is None:
-            negative_sampler_cls = BasicNegativeSampler
-
-        self.negative_sampler = negative_sampler_cls(
+        self.negative_sampler = negative_sampler_resolver.make(
+            query=negative_sampler_cls,
+            pos_kwargs=negative_sampler_kwargs,
```

Comment:
you sort of ruin them being pos_kwargs if you pass as kwargs 😆

Reply:
what about `pos_kwargs` = "Possibly kwargs"? 😉

Reply:
Positive kwargs?